catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -186,12 +168,23 @@ def make_property_helper(name, read_only=False):
         make_property_helper('offset_x')
         make_property_helper('offset_y')
 
+        make_property_helper('fan_status')
+        make_property_helper('cooler_mode')
+
         make_property_helper('sensor_width', read_only=True)
         make_property_helper('sensor_height', read_only=True)
 
         self.make_command('start_acquisition', self.start_acquisition)
         self.make_command('end_acquisition', self.end_acquisition)
 
+        self.critical_temp = self.config.get('critical_temp', 27.0)

```

Comment:
Let's also write thr local attribute out to "critical_temperature", like in the PR on thd-controls.

Reply:
done